src/app/error.tsx

```
@@ -1,35 +1,40 @@
 
-import Button from '@/shared/components/Button';
+import ButtonNewver from '@/shared/components/ButtonNewver';
 import PaddingWrapper from '@/shared/components/PaddingWrapper';
 import SadBbangleBox from '@/shared/components/SadBbangleBox';
 import { BbangleIcon } from '@/shared/components/icons';
 import PATH from '@/shared/constants/path';
 import Link from 'next/link';
 import { useEffect } from 'react';
+import DefaultLayout from '@/shared/components/DefaultLayout';
 
 const Error = ({ error, reset }: { error: Error & { digest?: string }; reset: () => void }) => {
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)